daemon/seccomp_disabled.go

```diff
@@ -0,0 +1,12 @@
+// +build !seccomp,!windows
```

Comment:
Now that I run this again, it looks like this change doesn't work either. It gives the following error.

```
// +build !seccomp

---> Making bundle: dynbinary (in bundles/1.11.0-dev/dynbinary)
Building: bundles/1.11.0-dev/dynbinary/docker-1.11.0-dev
# github.com/docker/docker/daemon
.gopath/src/github.com/docker/docker/daemon/seccomp_disabled.go:10: undefined: specs.LinuxRuntimeSpec
debian/rules:13: recipe for target 'override_dh_auto_build' failed
make[1]: *** [override_dh_auto_build] Error 1
```
Here are some other errors for some of the settings I tried.

```
// +build !seccomp !windows

Building: bundles/1.11.0-dev/dynbinary/docker-1.11.0-dev
# github.com/docker/docker/daemon
daemon/seccomp_disabled.go:10: undefined: specs.LinuxRuntimeSpec
daemon/seccomp_linux.go:15: setSeccomp redeclared in this block
	previous declaration at daemon/seccomp_disabled.go:10
make: *** [deb] Error 1
```

And here is the original error.
```
// +build !seccomp,!windows

Step 7 : RUN dpkg-buildpackage -uc -us
 ---> Running in d82a7feb7c3a
dpkg-buildpackage: source package docker-engine
dpkg-buildpackage: source version 1.11.0~dev~git20160317.014414.0.c32be45-0~jessie
dpkg-buildpackage: source distribution jessie
dpkg-buildpackage: source changed by Docker <support@docker.com>
 dpkg-source --before-build docker
dpkg-buildpackage: host architecture amd64
Unable to find image 'docker-temp/build-deb:debian-jessie' locally
Pulling repository docker.io/docker-temp/build-deb
docker: Error: image docker-temp/build-deb not found.
See 'docker run --help'.
tar: This does not look like a tar archive
tar: Exiting with failure status due to previous errors
---> Making bundle: .integration-daemon-stop (in bundles/1.11.0-dev/build-deb)
+++++ cat bundles/1.11.0-dev/build-deb/docker.pid
++++ kill 3033
++++ /etc/init.d/apparmor stop
Clearing AppArmor profiles cache:.
All profile caches have been cleared, but no profiles have been unloaded.
Unloading profiles will leave already running processes permanently
unconfined, which can lead to unexpected situations.

To set a process to complain mode, use the command line tool
'aa-complain'. To really tear down all profiles, run the init script
with the 'teardown' option."
make: *** [deb] Error 1
```

These errors where all from running ```make deb``` from this PR branch.

I'll update and run again and see if something was fixed since the last time i updated.